public/static/locales/cy/common.json

```diff
@@ -45,7 +45,7 @@
   "Feedback": "Adborth",
   "Congratulations, you've almost finished. We want to provide you with the best experience possible and value your feedback.": "Llongyfarchiadau, rydych chi bron â gorffen. Rydym ni eisiau rhoi'r profiad gorau posibl i chi a byddem yn croesawu eich adborth yn fawr.",
   "I agree to be contacted to provide feedback to help develop this service (optional)": "Rydw i'n cytuno y gallwch gysylltu â mi er mwyn holi am adborth er mwyn gallu datblygu'r gwasanaeth hwn (dewisol)",
-  "See here(/pdfs/feedback) for details on how we will use your data.": "Cymerwch gip yma (pdfs/adborth) i weld sut y byddwn yn defnyddio'ch data.",
+  "[How we will use your data (feedback pdf opens in new window)](/pdfs/feedback)": "[Sut y byddwn yn defnyddio'ch data. (adborth pdf yn agor mewn ffenestr newydd)] (/pdfs/adborth) ",
```

Comment:
Where did this translation come from?
We need to compile a list of statements that we need translations approved for the next release.